scripts/quick_install.sh

```diff
@@ -99,21 +94,62 @@ kubectl apply --filename=https://github.com/jetstack/cert-manager/releases/downl
 kubectl wait deployment.apps/cert-manager-webhook --namespace=cert-manager --for=condition=available --timeout=600s
 sleep 15
 
-# Install KFServing
+# Install Minio
+cat <<EOF > minio-values.yaml
```

Comment:
Can we move the existing `scripts/e2e/config/minio/values.yaml` file under `scripts/config` and reuse it here as well as the e2e?

Reply:
On a related note, why do we need this `quick_install.sh` and (`scripts/e2e/setup-cluster.sh` + `scripts/e2e/deploy-merlin.sh`)? Can we refactor them so that we use the same installation steps for the e2e and local dev?